.nvmrc

```diff
@@ -0,0 +1 @@
+16.20.2
```

Comment:
Got that from foundation, but is it ok if it's only v16 instead of v18?

Reply:
Good question